### PR TITLE
Update zh_CN.lang

### PR DESCRIPTION
**Pull request by Kiwi233**

### CODE REVIEW

File: src/main/resources/assets/neicustomdiagram/lang/zh_CN.lang [hunk 1]
```diff
@@ -1,22 +1,22 @@
 neicustomdiagram.api.stacksize=最大堆叠：%,d
 neicustomdiagram.api.hasnbt=有NBT(<Shift>查看)
-neicustomdiagram.api.cycleindex=构成：%,d / %,d
-neicustomdiagram.api.cycleindexwithshift=构成：%,d / %,d(<Shift>查看)
-neicustomdiagram.api.cyclecomponents=Cycle components:
+neicustomdiagram.api.cycleindex=循环索引：%,d / %,d
+neicustomdiagram.api.cycleindexwithshift=循环索引：%,d / %,d (<Shift>查看)
+neicustomdiagram.api.cyclecomponents=循环部件：
 neicustomdiagram.api.excesscyclecomponents=(+%,d more)
```
Comment (D-Cysteine) [inline src/main/resources/assets/neicustomdiagram/lang/zh_CN.lang:6]: I think you missed this line

File: src/main/resources/assets/neicustomdiagram/lang/zh_CN.lang [hunk 2]
```diff
@@ -126,11 +126,11 @@ neicustomdiagram.generators.gregtech5.materialparts.pipesslot=管道
 neicustomdiagram.generators.gregtech5.materialparts.specialpipesslot=特殊管道
 
 neicustomdiagram.generators.gregtech5.materialtools.groupname=GT材料工具
-neicustomdiagram.generators.gregtech5.materialtools.handlelabel=工具手柄
+neicustomdiagram.generators.gregtech5.materialtools.handlelabel=工具手柄材料：%s
```
Comment (D-Cysteine) [inline src/main/resources/assets/neicustomdiagram/lang/zh_CN.lang:129]: I think the left side is correct for this one; the usage of this string was changed:
ToolHandleTooltip

It no longer mentions the handle material

**Review by D-Cysteine — COMMENTED** [review]
Thanks again for the translation!

File: src/main/resources/assets/neicustomdiagram/lang/zh_CN.lang [hunk 3]
```diff
@@ -1,22 +1,22 @@
 neicustomdiagram.api.stacksize=最大堆叠：%,d
 neicustomdiagram.api.hasnbt=有NBT(<Shift>查看)
-neicustomdiagram.api.cycleindex=构成：%,d / %,d
-neicustomdiagram.api.cycleindexwithshift=构成：%,d / %,d(<Shift>查看)
-neicustomdiagram.api.cyclecomponents=Cycle components:
+neicustomdiagram.api.cycleindex=循环索引：%,d / %,d
+neicustomdiagram.api.cycleindexwithshift=循环索引：%,d / %,d (<Shift>查看)
```
Comment (GlodBlock) [inline src/main/resources/assets/neicustomdiagram/lang/zh_CN.lang:4]: 这两个索引应该叫页码

File: src/main/resources/assets/neicustomdiagram/lang/zh_CN.lang [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
+neicustomdiagram.api.cyclecomponents=循环部件：
```
Comment (GlodBlock) [inline src/main/resources/assets/neicustomdiagram/lang/zh_CN.lang:5]: 不是部件是工具

**Comment by Kiwi233** [comment]
Done @GlodBlock

**Review by GlodBlock — APPROVED** [review]

**Pull request merged**